Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -6249,6 +6249,28 @@
         }
       }
     },
+    "featuredAuthorCategoryEspanol" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Espanol"
```

Comment:
```suggestion
            "value" : "Español"
```

Reply:
I don't think we have the n with the noodles in the English alphabet.

Reply:
That's true, but this word doesn't exist in English -- it's in Spanish.

Reply:
You are right 😄